src/engine-scripts/puppet/loginReload.js

```diff
@@ -0,0 +1,10 @@
+const loginReload = async (page, scenario) => {
+    await page.goto(scenario.loginUrl);
```

Comment:
Was config.js scenarios edited to include the loginUrl?

Reply:
Ack. I messed that up.

Reply:
Fixed in latest patchset!